spec/models/miq_server/worker_management/monitor/kubernetes_spec.rb

```diff
@@ -101,7 +107,51 @@
     end
   end
 
-  context "#collect_initial_pods(private)" do
+  context "#save_deployment(private)" do
+    let(:pod_name) { "1-generic" }
+    let(:fake_deployment_data) do
+      RecursiveOpenStruct.new(
+        :metadata => {
+          :name => pod_name
+        },
+        :spec => {
+          :replicas => 2,
+          :template => {
+            :spec => {
+              :containers => [{:name => pod_name}]
+            }
+          }
+        },
+        :status => {
+          :readyReplicas => 2
+        }
+      )
+    end
+
+    it "saves replicas" do
+      server.send(:save_deployment, fake_deployment_data)
+      expect(server.current_deployments[pod_name].fetch_path(:spec, :replicas)).to eql(2)
```

Comment:
Should these `eq` rather than `eql`?

Reply:
Same with the others below.

Reply:
I always forget which is which since it doesn't matter 99% of the time.  🤣